includes/connect/class-wc-stripe-connect.php

```
@@ -368,6 +373,47 @@ private function save_stripe_keys( $result, $type = 'connect', $mode = 'live' )
 
+		/**
+		 * Persists the test-mode keys returned alongside a live Connect OAuth result.
+		 *
+		 * @param object $result OAuth result that may carry testPublishableKey/testSecretKey.
+		 */
+		private function save_oauth_test_keys_alongside_live( $result ): void {
+			if ( ! $result instanceof stdClass || ! isset( $result->testPublishableKey, $result->testSecretKey ) ) { // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+				return;
+			}
+
+			$settings = WC_Stripe_Helper::get_stripe_settings();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'save_oauth_test_keys_alongside_live'. (undefined)

(StaticAccess)